lib/swagger.js

```diff
@@ -21,37 +21,64 @@ exports.run = function({ args, opts }) {
     return Promise.reject(new Error('No api key provided. Please use --key'));
   }
 
-  if (!args[0]) {
-    return Promise.reject(
-      new Error('No swagger file provided. Usage `rdme swagger <swagger-file>`'),
-    );
-  }
+  function callApi(specPath) {
+    function success(data) {
+      console.log(data);
+      console.log('Success! '.green);
+    }
 
-  function success(data) {
-    console.log(data);
-    console.log('Success! '.green);
-  }
+    function error(err) {
+      try {
+        if (err.statusCode === 401) {
+          return Promise.reject(
+            new Error('There was a problem with your api key. Please try again.'),
```

Comment:
I've actually added this in the readme http api, you can see it over in #1553 over there! So we probably don't need this anymore.

Reply:
I'll pull it here and we can just rely on that change when that gets deployed.